tests/sentry/feedback/test_feedback_ingest.py

```diff
@@ -0,0 +1,136 @@
+        },
+    },
+}
+
+wrong_type_test_data = {
+    "timestamp": "1694039635.9195",
+    "message": 24,
+    "event_id": "b51647a3c56f4a939984bb1147a6c3e5",
+}
+
+
+class FeedbackIngestTest(MonitorIngestTestCase):
+    endpoint = "sentry-api-0-feedback-ingest"
+
+    def test_save_feedback(self):
+        # Feature enabled should lead to successful save
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path, data=test_data, **self.dsn_auth_headers)
+            assert response.status_code == 201
+
+            # Feedback object exists
+            feedback_list = Feedback.objects.all()
+            assert len(feedback_list) == 1
+
+            # Feedback object is what was posted
+            assert feedback_list[0].data["type"] == "transaction"
+            assert feedback_list[0].data["environment"] == "prod"
+            assert feedback_list[0].message == "This website is great!"
+
+    def test_no_feature_enabled(self):
+        # Feature disabled should lead to unsuccessful save
+        with self.feature({"organizations:user-feedback-ingest": False}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path, data=test_data, **self.dsn_auth_headers)
+            assert response.status_code == 404
+
+    def test_not_authorized(self):
+        # No authorization should lead to unsuccessful save
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path, data=test_data)
+            assert response.status_code == 401
+            assert response.data == {"detail": "Authentication credentials were not provided."}
+
+    def test_wrong_input(self):
+        # Wrong inputs should lead to failed validation
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path, data=wrong_test_data, **self.dsn_auth_headers)
+            assert response.status_code == 500
+
+    def test_no_event_id(self):
+        # Event ID is required for a successful post
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(
+                path, data=missing_event_id_test_data, **self.dsn_auth_headers
+            )
+            assert response.status_code == 400
+
+    def test_wrong_type(self):
+        # Fields should be correct type
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path, data=wrong_type_test_data, **self.dsn_auth_headers)
+            assert response.status_code == 500
```

Comment:
i don't think we want to 500 in this case -- 500 usually signifies an unhandled error. we should be raising something like a `400` in this case.